src/lib/AnnotationControls.ts

```diff
@@ -46,6 +50,7 @@ export default class AnnotationControls {
     public destroy(): void {
         fullscreen.removeListener('enter', this.handleFullscreenEnter);
         fullscreen.removeListener('exit', this.handleFullscreenExit);
+        this.annotator.removeListener('annotationcreate', this.deactivateCurrentControl);
```

Comment:
We're going to need to do several things when this event fires, so having it in a more central place is probably preferable.